src/conf.py

```diff
@@ -308,22 +250,19 @@
 
 # Additional stuff for the LaTeX preamble.
 latex_elements['preamble'] = '\\usepackage{amsmath}\n\\usepackage{amssymb}\n\\usepackage[retainorgcmds]{IEEEtrantools}\n'
-
-imgmath_image_format='svg'
+imgmath_image_format='png'
```

Comment:
I would have kept `svg`. Why did you change?

Reply:
This is a residual setting that i missed when removing the imgmath extension